### DISCUSSION TITLE
Change `cycfi` account type to organization

### DISCUSSION THREAD

**Discussion by paulocoutinhox** [cycfi/elements#353] [Ideas]
As a great library i suggest you create an organization in github to put this great project.

It will be more professional. Something like:
github.com/elements/elements

Thanks for the great work.

**Comment by djowel** [comment]
I'm not familiar with github organizations. What advantages does it have? Why is it more "professional"?

**Comment by redtide** [comment]
It's mainly about user access levels and team management, it is described [here].
So if there would be any need for that I suppose it should be a matter to change the cycfi account type.
Here [how to change an user account to an organization].

[here]: https://docs.github.com/en/github/getting-started-with-github/types-of-github-accounts
[how to change an user account to an organization]: https://docs.github.com/en/github/setting-up-and-managing-your-github-user-account/converting-a-user-into-an-organization

**Comment by redtide** [comment]
I changed the OP title, which would like to move elements in an `elements` organization account type, which doesn't seem to help because it would create a fragment separated from other projects, or I'm missing something about the benefit of doing so.
Note that by doing this the account cannot be used anymore to commit or do other actions as it was an user as is now, it's somewhat like a level up that can manage multiple users with rules and authorizations.
